test/ambient.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.json' {
```

Comment:
This allows for importing `.json` files in tests, like:

```ts
import pkg from '../package.json'
```

Reply:
nice!